### DISCUSSION TITLE
Create modal

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5480] [Q&A]
How can i make button in any table for example : Send notification , and when i click on it popup a modal with fields then fill it and send or post anything. Please is it available to do that ? Necessary

**Comment by edalzell** [comment]
Sorry, I don't understand your question, can you please explain in more detail what you're trying to do?

**Reply by HazemMohamed252** [reply]
For example in users view the table display email, name, roles and last login, I want to add new column with button call : Send Message and when i click on it it pop up a modal with title and description and send button to send a message to this user. I hope this be clear for you. Thanks

**Comment by edalzell** [comment]
I don't think actions can have prompts like that, though that's a decent feature request.

You could make your own listing view, then do whatever you like though, would that work?

**Comment by jasonvarga** [comment]
You can create an action with fields. The modal that is displayed will let you type into any kind of field you want.

https://statamic.dev/extending/actions#adding-fields

**Reply by edalzell** [reply]
oh I did not know this, thanks Jason
